tests/framework/ROM/TimeSeries/ARMA/MultipleFourier/dataGenerator.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# Number of samples (units of "t")
+N = 100
+# pivot param
+t = np.arange(N)
+
+A = np.zeros(N)
+B = np.zeros(N)
+C = np.zeros(N)
+D = np.zeros(N)
+
+# Fourer Moments: A sin(2pi/k * t)
+# A and B share the same frequencies, as do C and D
+f_A = {100.0 : 1.0,
+        20.0 : 1.0}
+f_B = {100.0 : 4.0,
+        20.0 : 2.0}
+f_C = { 50.0 : 1.0,
+        10.0 : 1.0}
+f_D = { 50.0 : 4.0,
```

Comment:
I think you need to follow the raven standards about the variables.

Reply:
If so, please update this file

Reply:
While this standard has NOT been propagated to scripts outside the framework, I can make these changes.  This should either be standardized or not required as part of the merging review.

Reply:
I do not think we should enforce a standard in the external model used for testing. 
Another story would be if we talk about Plugins (in there we should enforce a standard if they want to borrow our NQ1 level/docs)

Reply:
but in case it is a script generated by us and it might be useful to keep our standards and degree of docstrings/description